### PR TITLE
EZP-26756: FieldRelation is not handled in Solr Location search

### PR DESCRIPTION
**Pull request by pspanja**
This PR resolves https://jira.ez.no/browse/EZP-26756
Related PR: https://github.com/ezsystems/ezplatform-solr-search-engine/pull/80

FieldRelation is not handled in Location search when using Solr search engine.

### CODE REVIEW

**Comment by pspanja** [comment]
This does not fail because we convert `NotImplementedException` to skipped test.
Will try to make that controlled from the test.

**Comment by andrerom** [comment]
> This does not fail because we convert NotImplementedException to skipped test.

yes, this is an issue actually.. it can hide bugs

**Comment by bdunogier** [comment]
> This will require Solr engine v1.2

You mean that solr integration tests will only pass on kernel with solr 1.2, do you ?

**Comment by pspanja** [comment]
@bdunogier yes

**Comment by pspanja** [comment]
I removed cleanup of search integration tests to be opened separately (after this is merged).
This now targets 6.4.

File: bin/.travis/prepare_unittest.sh [hunk 1]
```diff
@@ -35,8 +35,10 @@ COMPOSER_UPDATE=""
 # solr package search API integration tests
 if [ "$TEST_CONFIG" = "phpunit-integration-legacy-solr.xml" ] ; then
     echo "> Require ezsystems/ezplatform-solr-search-engine:^1.0.0@dev"
-    composer require --no-update ezsystems/ezplatform-solr-search-engine:^1.0.0@dev
+    composer require --no-update ezsystems/ezplatform-solr-search-engine dev-EZP-26756-fieldrelation-location-search@dev
```
Comment (andrerom) [inline bin/.travis/prepare_unittest.sh]: tmp

**Comment by pspanja** [comment]
Tested locally that the tests are not skipped, integration test cleanup coming as a follow-up.

> +1, even if group: deprecated is supposed to be unneeded in the next 12 hours if fix really fixes it.

About to find out...

**Pull request merged**